### ISSUE TITLE
Raw event logging from payments-server

### ISSUE THREAD

**Issue by data-sync-user** [mozilla/fxa#4652]
For the new metric pipeline, we are moving event transformation out of the web code (where all the transforms are currently done for the logged events), and will be handling them in a new metrics worker. To allow this work to proceed in parallel we want to log the raw events while continuing the existing output.

The raw event logging option should be behind a feature/config flag so that it can be turned on when we have more of the pipeline in place to process them.

Verification must also be done to ensure the existing log processing code correctly ignores these raw events.

┆Issue is synchronized with this [Jira Task](https://jira.mozilla.com/browse/FXA-1410)
┆Issue Number: FXA-1410

**Comment by data-sync-user** [comment]
➤ Barry Chen commented:

This is a duplicate of FXA-1445.
